lib/uwb/include/uwb/UwbRegisteredCallbacks.hxx

```diff
@@ -17,6 +17,10 @@ namespace uwb
  */
 class RegisteredCallbackToken;
 
+class RegisteredSessionCallbackToken;
+
+class RegisteredDeviceCallbackToken;
+
```

Comment:
Let's remove these since they're internal details and so the interface clients shouldn't be aware of them.
```suggestion
```

Reply:
I pushed a change implementing this, but I'm wondering if it even makes sense to have one deregistration function, while there are two registration functions. I'll think about this more when I make a pr calling the deregistration function where it's needed.

Reply:
Having 2 deregistration functions seems reasonable to me.